packages/react-app/src/views/HomeView.jsx

```diff
@@ -162,29 +162,30 @@ export default function HomeView({ connectedBuilder, userProvider }) {
           </Text>
         </Container>
 
-        <Box
-          bgImg="/assets/header_platform.svg"
-          backgroundRepeat="repeat-x"
-          backgroundSize="auto 130px"
-          h="130px"
-          mb="35px"
-        />
-
-        <Container maxW="container.lg" centerContent>
-          <Box>
-            {Object.entries(challengeInfo).map(([challengeId, challenge], index) => (
-              <ChallengeExpandedCard
-                challengeId={challengeId}
-                challenge={challenge}
-                challengeIndex={index}
-                builderAttemptedChallenges={builderAttemptedChallenges}
-                userProvider={userProvider}
-                connectedBuilder={connectedBuilder}
-              />
-            ))}
-          </Box>
-        </Container>
+        <Box bgImg="/assets/header_platform.svg" backgroundRepeat="repeat-x" backgroundSize="auto 130px" h="130px" />
       </Box>
+      {Object.entries(challengeInfo).map(([challengeId, challenge], index) =>
+        index === 0 ? (
+          <ChallengeExpandedCard
+            challengeId={challengeId}
+            challenge={challenge}
+            challengeIndex={index}
+            builderAttemptedChallenges={builderAttemptedChallenges}
+            userProvider={userProvider}
+            connectedBuilder={connectedBuilder}
+            isFirst
+          />
+        ) : (
+          <ChallengeExpandedCard
+            challengeId={challengeId}
+            challenge={challenge}
+            challengeIndex={index}
+            builderAttemptedChallenges={builderAttemptedChallenges}
+            userProvider={userProvider}
+            connectedBuilder={connectedBuilder}
+          />
+        ),
+      )}
```

Comment:
This can be cleaner (without the condition) if you pass the prop as:

```js
<ChallengeExpandedCard
    challengeId={challengeId}
    challenge={challenge}
    challengeIndex={index}
    builderAttemptedChallenges={builderAttemptedChallenges}
    userProvider={userProvider}
    connectedBuilder={connectedBuilder}
    isFirst={index === 0}
  />
```

Reply:
lol, I overthought the solution, updated it